### PR TITLE
Ci 2 Migration

### PR DESCRIPTION
**Pull request by Darshan-Chauhan**
Hi Team,
This PR includes
- CI 2 migration
- Android upgrade to 6.3.0
- Enables push notifications for Android phones

Please review it and let me know if any further changes are required.
Thanks

### CODE REVIEW

File: app/services/cordova.js [hunk 1]
```diff
@@ -22,59 +22,57 @@ export default Ember.Service.extend({
   },
 
   initiatePushNotifications() {
-    if(this.isIOS()) {
```
Comment (Darshan-Chauhan) [inline app/services/cordova.js]: Only if condition for IOS is removed nothing else

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+        - upload-coverage:
+            requires:
+              - package_dependencies
+              - test_cases
+            filters:
+              branches:
+                only: /^(master|live)$/
+        - www_build:
+            requires:
+              - upload-coverage
```
Comment (steveyken) [inline .circleci/config.yml]: I would switch this to require `test_cases` as upload_coverage isn't a hard dependency.

```suggestion
              - test_cases
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+                only: /^(master|live)$/
+        - www-deploy:
+            requires:
+              - www_build
+            filters:
+              branches:
+                only: /^(master|live)$/
+        - ember_cordova_build:
+            requires:
+              - www-deploy
```
Comment (steveyken) [inline .circleci/config.yml]:
```suggestion
              - test_cases
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+          name: Setup Environment variables
+          command: |
+            echo 'export PATH=$PATH:`yarn bin`' >> $BASH_ENV
+            echo 'export ENVIRONMENT=$(if [ "$CIRCLE_BRANCH" == "live" ]; then echo production; else echo staging; fi)' >> $BASH_ENV
+            echo 'export PROVISIONING_PROFILE=$(if [ "$CIRCLE_BRANCH" == "live" ]; then echo $ENCODED_PROD_PROVISIONING_PROFILE; else echo $ENCODED_STAGING_PROVISIONING_PROFILE; fi)' >> $BASH_ENV
+            echo 'export CERTIFICATES=$(if [ "$CIRCLE_BRANCH" == "live" ]; then echo $ENCODED_PROD_CERT; else echo $ENCODED_STAGING_CERT; fi)' >> $BASH_ENV
+      - run:
+          command: |
+              npm rebuild node-sass
+              yarn run ember build --environment=production
```
[the remark this answers is not in the document]
Comment (Darshan-Chauhan) [reply .circleci/config.yml]: Nothing from previous steps is available for `ios build` as it's totally different shell. So I'd to make the build again

**Comment by Darshan-Chauhan** [comment]
@steveyken , addressed your concerns.

**Pull request merged**